shared/src/main/scala/org/yaml/render/JsonRender.scala

```diff
@@ -133,7 +132,7 @@ class JsonRender[W: Output] private (private val writer: W,
       case Float =>
         val s = scalar.value.toString
         if (s.indexOf('.') == -1 && !s.contains('e') && !s.contains('E')) s + ".0" else s // Bug in scala-js toString
-      case Null => "null"
+      case Null => ""
```

Comment:
Nevermind, it was fixed on ALS. but the main cause is that for YNode.Empty, the YType is null (instead of Empty, for example), this causing some unwanted characters on completion. Needless to say, that is likely a breaking change that is out of the scope of the issue I was trying to fix.